src/utils/errors.ts

```diff
@@ -24,8 +24,11 @@ export enum LifiErrorCode {
   SlippageError = 1011,
   TransactionRejected = 1012,
   BalanceError = 1013,
+  AllowanceRequired = 1014,
 }
 
+export const ERC20AllowanceError = 'ERC20: transfer amount exceeds allowance'
+
```

Comment:
Please remove this and make it inline like the rest of the error strings.

Reply:
I thought of keeping all raw strings in consts rather than using directly.

1. Reduces typos
2. Increases readability.

Any particular reason for using inline?